opennms-model/src/main/java/org/opennms/netmgt/model/events/NodeLabelChangedEventBuilder.java

```diff
@@ -0,0 +1,79 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2018 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2018 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.netmgt.model.events;
+
+import java.util.Date;
+
+import org.opennms.netmgt.events.api.EventConstants;
+import org.opennms.netmgt.xml.event.Event;
+import org.opennms.netmgt.xml.event.Parm;
+
+import com.google.common.base.Strings;
+
+public class NodeLabelChangedEventBuilder extends EventBuilder {
+
+    public NodeLabelChangedEventBuilder(final String source) {
+        super(EventConstants.NODE_LABEL_CHANGED_EVENT_UEI, source, new Date());
```

Comment:
Does this not trigger the `checkForIllegalUei()` check defined in the `EventBuilder` class?

Reply:
you are right. I fixed it by overriding the method in the subclass